src/LinuxTracing/UprobesUnwindingVisitor.h

```diff
@@ -117,6 +121,9 @@ class UprobesUnwindingVisitor : public PerfEventVisitor {
 
   void SendFullAddressInfoToListener(const unwindstack::FrameData& libunwindstack_frame);
 
+  template <typename StackPerfEventDataT>
+  bool UnwindStack(const StackPerfEventDataT& event, orbit_grpc_protos::Callstack* callstack);
```

Comment:
Consider a name that specifies more that `callstack` is an out parameter, e.g., `resulting_callstack`.

Reply:
Ah good point. I also wanted to change that. Done.